charts/covidDataExplorer/CovidDataExplorer.tsx

```diff
@@ -867,6 +868,9 @@ export class CovidDataExplorer extends React.Component<{
 
         chartProps.map.targetYear = undefined
         chartProps.map.variableId = this.yColumn.spec.owidVariableId
+
+        // Preserve region
```

Comment:
Region is not preserved in explorer currently. Down the line I've got a better way to merge chart props.